entrypoint/selinux/selinux_suite_test.go

```diff
@@ -0,0 +1,46 @@
+package selinux
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/rancher/distros-test-framework/config"
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var cfg *config.ProductConfig
+
+func TestMain(m *testing.M) {
+	var err error
+
+	flag.Var(&customflag.ServiceFlag.InstallMode, "installVersionOrCommit", "Install upgrade customflag for version bump")
+	flag.Var(&customflag.ServiceFlag.ClusterConfig.Destroy, "destroy", "Destroy cluster after test")
+	flag.Var(&customflag.ServiceFlag.Channel, "channel", "channel to use on install or upgrade")
+	flag.Parse()
+
+	cfg, err = config.AddConfigEnv("../../config")
```

Comment:
here you need to refactor to same as https://github.com/bguzman-3pillar/distros-test-framework/blob/f468580ee4c3b8f467bfeae8462fe1d39d1de0e3/entrypoint/createcluster/createcluster_suite_test.go#L24-L25

Reply:
deleteWorkload bool